src/pages/welcome/WelcomePage.css

```diff
@@ -0,0 +1,42 @@
+.home-column-left {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
```

Comment:
You shouldn't be using hardcoded colors, you should create a constant and use it from there